tensorboard/uploader/uploader_subcommand.py

```diff
@@ -178,47 +178,54 @@ class _DeleteExperimentIntent(_Intent):
 
     _MESSAGE_TEMPLATE = textwrap.dedent(
         """\
-        This will delete the experiment on https://tensorboard.dev with the
-        following experiment ID:
+        This will delete the {num} experiments on
+        https://tensorboard.dev with the following experiment ID:
 
-        {experiment_id}
+        {experiment_id_list}
 
         You have chosen to delete an experiment. All experiments uploaded
         to TensorBoard.dev are publicly visible. Do not upload sensitive
         data.
         """
     )
 
-    def __init__(self, experiment_id):
-        self.experiment_id = experiment_id
+    def __init__(self, experiment_id_list):
+        self.experiment_id_list = experiment_id_list
 
     def get_ack_message_body(self):
-        return self._MESSAGE_TEMPLATE.format(experiment_id=self.experiment_id)
+        return self._MESSAGE_TEMPLATE.format(
+            num=len(self.experiment_id_list),
+            experiment_id_list=self.experiment_id_list,
+        )
 
     def execute(self, server_info, channel):
         api_client = write_service_pb2_grpc.TensorBoardWriterServiceStub(
             channel
         )
-        experiment_id = self.experiment_id
-        if not experiment_id:
+        if not self.experiment_id_list:
             raise base_plugin.FlagsError(
-                "Must specify a non-empty experiment ID to delete."
-            )
-        try:
-            uploader_lib.delete_experiment(api_client, experiment_id)
-        except uploader_lib.ExperimentNotFoundError:
-            _die(
-                "No such experiment %s. Either it never existed or it has "
-                "already been deleted." % experiment_id
-            )
-        except uploader_lib.PermissionDeniedError:
-            _die(
-                "Cannot delete experiment %s because it is owned by a "
-                "different user." % experiment_id
+                "Must specify at least one experiment ID to delete."
             )
-        except grpc.RpcError as e:
-            _die("Internal error deleting experiment: %s" % e)
-        print("Deleted experiment %s." % experiment_id)
+        for experiment_id in self.experiment_id_list:
+            if not experiment_id:
+                raise base_plugin.FlagsError(
+                    "Must specify a non-empty experiment ID to delete."
+                )
+            try:
+                uploader_lib.delete_experiment(api_client, experiment_id)
+                print("Deleted experiment %s." % experiment_id)
+            except uploader_lib.ExperimentNotFoundError:
+                _die(
```

Comment:
Add a test for errors while deleting a list of experiments? Specifically encode in the test your choice about whether to continue deleting experiments or whether to stop deleting.